src/main/kotlin/br/unipar/plano/interfaces/rest/solicitacaoprocedimento/SolicitacaoProcedimentoResource.kt

```diff
@@ -0,0 +1,56 @@
+package br.unipar.plano.interfaces.rest.solicitacaoprocedimento
+
+import br.unipar.plano.domain.solicitacaoprocedimento.model.IdSolicitacaoProcedimento
+import br.unipar.plano.domain.solicitacaoprocedimento.service.SolicitacaoProcedimentoService
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder
+import java.util.*
+import javax.validation.Valid
+
+@RestController
+@RequestMapping("/solicitacao-liberacao-procedimento")
+class SolicitacaoProcedimentoResource(val solicitacaoProcedimentoService: SolicitacaoProcedimentoService) {
+
+    @PostMapping("/criar")
+    @ResponseStatus(code = HttpStatus.CREATED)
+    fun insert(@RequestBody @Valid solicitacaoProcedimentoDTO: SolicitacaoProcedimentoDTO): ResponseEntity<Void> {
+        val idSolicitacaoProcedimento = solicitacaoProcedimentoService.insert(solicitacaoProcedimentoDTO)
+
+        val uri = ServletUriComponentsBuilder.fromCurrentRequest()
+            .path("/{id}")
+            .buildAndExpand(idSolicitacaoProcedimento.id)
+            .toUri()
+
+        return ResponseEntity.created(uri).build();
+    }
+
+    @PutMapping("/liberar/{idSolicitacaoProcedimento}")
```

Comment:
Mesma observação de nunca colocar verbo na url, no caso você está fazendo uma nova liberação para um procedimento especifico e não alterando uma.

```suggestion
    @PostMapping("/{idSolicitacaoProcedimento}/liberacao")
```